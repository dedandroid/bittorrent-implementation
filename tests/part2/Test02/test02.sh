#!/bin/bash
i='2'

cd ..
./peer1 Test0$i/test0$i.in > Test0$i/test0$i.out
cd Test0$i
diff test0$i.out test0$i.expected
if [ $? -ne 0 ]; then
	echo "Outputs don't match!"
else
	echo "Outputs match!"
fi
