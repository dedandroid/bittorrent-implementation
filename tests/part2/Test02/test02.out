Unable to connect to request peer
Not connected to any peers
